l2tdevtools/build_helpers/dpkg.py

```diff
@@ -487,9 +505,9 @@ def Build(self, source_helper_object):
     Returns:
       bool: True if successful, False otherwise.
     """
-    source_filename = source_helper_object.Download()
+    source_filename = source_helper_object.GetSourcePackageFilename()
     if not source_filename:
-      logging.info('Download of: {0:s} failed'.format(
+      logging.info('Missing source package of: {0:s}'.format(
```

Comment:
In all these log messages, please include the source_filename, to speed debugging.

Reply:
I cannot here, since there is no source_filename see line 500